app/views/govuk_component/docs/metadata.yml

```diff
@@ -17,6 +17,15 @@ fixtures:
     from:
```

Comment:
Have removed the first one - there was already an example of a non-truncated 'from' in the page.